mta-sts.md

```diff
@@ -279,24 +235,24 @@ and TXT endpoints are updated and the TXT record's TTL has passed.
 
 ## Policy Discovery & Authentication
 
-Senders discover a recipient domain's STS policy, by making an attempt to fetch
-TXT records from the recipient domain's DNS zone with the name "_mta_sts". A
-valid TXT record presence in "_mta_sts.example.com" indicates that the recipent
-domain supports STS.  To allow recipient domains to safely serve new policies,
-it is important that senders are able to authenticate a new policy retrieved for
-a recipient domain.
-
-Web PKI is the mechanism used for policy authentication. In this mechanism, the
-sender fetches a HTTPS resource (policy) from a host at `policy.mta-sts` in the
-Policy Domain. The policy is served from a "well known" URI:
-`https://policy.mta-sts.example.com/.well-known/mta-sts/current`. To consider 
-the policy as valid, the `policy_id` field in the policy MUST match the `id` 
-field in the DNS TXT record under `_mta_sts`.
-
-When fetching a new policy or updating a policy, the new policy MUST be
-fully authenticated (HTTPS certificate validation + peer verification) before
-use.  A policy which has not ever been successfully authenticated MUST NOT be
-used to reject mail.
+Senders discover a recipient domain's MTA-STS policy by fetching a TXT record
+from the recipient domain's DNS zone with the name `_mta_sts`. A valid TXT
```

Comment:
I was confused here that a TXT record merely had to exist. Maybe because "validity" is defined elsewhere